src/bin/sol-fbp-generator/main.c

```diff
@@ -1886,6 +1886,9 @@ print_usage(const char *program)
         "    -j  When resolving types, use the passed DESC files. If DESC is\n"
         "        a directory then all the .json files in the directory will be used.\n"
         "        Multiple -j can be passed.\n"
+        "        Note that Soletta install dir will also be looked, as well as any\n"
```

Comment:
The problem with this (always looking at a predefined set of paths) is that if I don't want to look to Soletta install dir. For example, I'm using sol-fbp-generator in my host machine to generate flows targetting Soletta on a non-Linux system with a different set of nodes available.